docker/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.5"
+ 
+services:
+  lesmevesajudes-ui:
+    image: "node:9"
```

Comment:
Why not nodejs 11 which is current?

Reply:
If you think 11 will work, ok, I'll change.

Reply:
Yes it works.